optimum/commands/onnxruntime/quantize.py

```diff
@@ -0,0 +1,60 @@
+from argparse import ArgumentParser
+from pathlib import Path
+
+from ...onnxruntime.configuration import AutoQuantizationConfig
+from ...onnxruntime.quantization import ORTQuantizer
+
+
+def parse_args_onnxruntime_quantize(parser):
+    required_group = parser.add_argument_group("Required arguments")
+    required_group.add_argument(
+        "--onnx_model", type=Path, required=True, help="Path indicating where the ONNX models to quantize are located."
+    )
+
+    optional_group = parser.add_argument_group("Optional arguments")
+    optional_group.add_argument(
+        "-o",
+        "--output",
+        type=Path,
+        help="Path indicating the directory where to store generated ONNX model. (defaults to --onnx_model value).",
+    )
+
+    level_group = parser.add_mutually_exclusive_group(required=True)
+    level_group.add_argument("--arm64", action="store_true", help="Quantization for the ARM64 architecture.")
+    level_group.add_argument("--avx2", action="store_true", help="Quantization with AVX-2 instructions.")
+    level_group.add_argument("--avx512", action="store_true", help="Quantization with AVX-512 instructions.")
+    level_group.add_argument(
+        "--avx512_vnni", action="store_true", help="Quantization with AVX-512 and VNNI instructions."
+    )
+    level_group.add_argument("--tensorrt", action="store_true", help="Quantization for NVIDIA TensorRT optimizer.")
+
+
+class ONNXRuntimmeQuantizeCommand:
+    def __init__(self, args):
+        self.args = args
+
+    def run(self):
+        if not self.args.output:
+            save_dir = self.args.onnx_model
+        else:
+            save_dir = self.args.output
+
+        quantizers = []
+
+        quantizers = [
+            ORTQuantizer.from_pretrained(save_dir, file_name=model.name) for model in save_dir.glob("*.onnx")
+        ]
+
+        if self.args.arm64:
+            dqconfig = AutoQuantizationConfig.arm64(is_static=False, per_channel=False)
+        elif self.args.avx2:
+            dqconfig = AutoQuantizationConfig.avx2(is_static=False, per_channel=False)
+        elif self.args.avx512:
+            dqconfig = AutoQuantizationConfig.avx512(is_static=False, per_channel=False)
+        elif self.args.avx512_vnni:
+            dqconfig = AutoQuantizationConfig.avx512_vnni(is_static=False, per_channel=False)
```

Comment:
Could you add a `--per-channel` argument? We could default to `per_channel=False` but we may want to allow it here.